examples/proc_GDS_capture_int.py

```diff
@@ -0,0 +1,108 @@
+import pyspecdata as psd
+import matplotlib.pyplot as plt
+import numpy as np
+from itertools import cycle
+
+colorcyc_list = plt.rcParams["axes.prop_cycle"].by_key()["color"]
+color_cycle = cycle(colorcyc_list)
+
+V_atten_ratio = 102.35  # attenutation ratio
+HH_width = 2e6
+int_slop = 1e-6
+
+with psd.figlist_var() as fl:
+    for filename, nodename in [
+        (
+            "240819_amp0p05_beta_max_pulse_capture.h5",
+            "pulse_capture_9",
+        ),
+        # (
+        #    "240819_amp0p1_beta_max_pulse_capture.h5",
+        #    "pulse_capture_6",
+        # ),
+    ]:
+        s = psd.find_file(
+            filename, expno=nodename, exp_type="ODNP_NMR_comp/test_equipment"
+        )
+        amplitude = s.get_prop("acq_params")["amplitude"]
+        beta_us_sqrt_W = s.get_prop("acq_params")["beta_90_s_sqrtW"]
+        fl.basename = (
+            f"amplitude = {amplitude}, $\\beta$ = {beta_us_sqrt_W} \n"
+        )
+        if not s.get_units("t") == "s":
+            print(
+                "units weren't set for the t axis or else I can't read them from the hdf5 file!"
+            )
+            s.set_units("t", "s")
+        s *= V_atten_ratio  # attenutation ratio
+        s /= np.sqrt(50)  # V/sqrt(R) = sqrt(P)
+        fl.next(r"$\sqrt{P_{analytic}}$ vs $t_{pulse}$")
+        raw_color = next(color_cycle)
+        fl.plot(s, color=raw_color, label="raw analytic")
+        abs_color = next(color_cycle)
+        fl.plot(abs(s), color=abs_color, label="abs(analytic)")
+        # {{{ apply frequency filter
+        SW = 1 / (s["t"][1] - s["t"][0])  # nyquists frequency
+        carrier = (
+            s.get_prop("acq_params")["carrierFreq_MHz"] * 1e6
+        )  # signal frequency
+        n = np.floor(
+            (carrier + SW / 2) / SW
+        )  # nearest integer multiple of sampling frequency
+        nu_a = carrier - n * SW  # aliasing frequency
+        if nu_a < 0:
+            perceived_nu = abs(
+                nu_a
```

Comment:
This is not explained.  If you are claiming you want the abs, you need to explain why.  Why is the negative peak located here?

Reply:
Also `nu_a` is the perceived frequency, so the multiple variables flying around are confusing